src/System.Private.CoreLib/shared/System/TimeZoneInfo.Unix.cs

```diff
@@ -100,13 +101,13 @@ private TimeZoneInfo(byte[] data, string id, bool dstDisabled)
             if (!dstDisabled)
             {
                 // only create the adjustment rule if DST is enabled
-                TZif_GenerateAdjustmentRules(out _adjustmentRules, _baseUtcOffset, dts, typeOfLocalTime, transitionType, StandardTime, GmtTime, futureTransitionsPosixFormat);
+                TZif_GenerateAdjustmentRules(out _adjustmentRules, _baseUtcOffset, dts, typeOfLocalTime, transitionType!, StandardTime, GmtTime, futureTransitionsPosixFormat);
```

Comment:
Why is this `!` needed? Neither the target parameter nor the local are nullable.

Reply:
It's not; removed.  It's a holdover from an interim edit and I never removed it.  Looking forward to an analyzer for finding and fixing such unnecessary usage.